app/tests/src/db/models/factories.py

```diff
@@ -69,6 +70,8 @@ class Meta:
 
     name = factory.Faker("word")
     content = factory.Faker("text")
+    program = random.choice([None, "SNAP", "Medicaid", "TANF"])
+    region = random.choice([None, "MI", "MD", "PA"])
```

Comment:
You'll want to use `factory.Faker('random_choices', elements=['XYZ', 'ABC', '123', '456'])` here -- random.choice will be evaluated once as Python reads the class definition, so every subsequent call to the factory will have the same program/region.

nit: may want to make this consistent with the example usage ("guru-snap", "Michigan"). If you wanted, you could enforce this at the DB level by making these enum types (example [here](https://github.com/navapbc/template-application-flask/blob/ecc0dd050348632f0ac641581d075edb703303f2/app/src/db/models/user_models.py#L51))

Reply:
Good catch! Thanks for the solution!

Reply:
> If you wanted, you could enforce this at the DB level by making these enum types (example [here](https://github.com/navapbc/template-application-flask/blob/ecc0dd050348632f0ac641581d075edb703303f2/app/src/db/models/user_models.py#L51))

Don't think we want to use enum since we're allowing `program` and `region` values to be specified on the command line and be flexible to new values without redeployment, so keeping it as `Text`.